xsnippet_api/services/snippet.py

```diff
@@ -0,0 +1,63 @@
+"""
+    xsnippet_api.services.snippet
+    -----------------------------
+
+    Snippet service implements domain business logic for managing
+    snippets. Its purpose is to avoid writing business logic in
+    RESTful API.
+
+    :copyright: (c) 2016 The XSnippet Team, see AUTHORS for details
+    :license: MIT, see LICENSE for details
+"""
+
+import datetime
+
+from aiohttp import web
+import pymongo
+
+
+class Snippet:
+
+    def __init__(self, db):
+        self.db = db
+
+    async def create(self, snippet):
+        snippet = self._normalize(snippet)
+        await self.db.snippets.insert(snippet)
+        snippet['id'] = snippet.pop('_id')
+        return snippet
+
+    async def get(self, *, limit=100):
+        snippets = await self.db.snippets.find()\
+            .sort('_id', pymongo.DESCENDING)\
```

Comment:
Sorting by ``_id`` is a side-effect of having auto-incrementing IDs, which may or may not change in the future :P 

Maybe just sort by ``created_at``?